dependency-check-maven-config.xml

```diff
@@ -100,4 +100,15 @@
         <cve>CVE-2018-11788</cve>
     </suppress>
 
+    <suppress>
+        <notes><![CDATA[
+   file name: lodash-2.4.1.jar
```

Comment:
I would put in these details:

We're safe to suppress this vulnerability, I've checked into it.  We have 3 dependencies that use it:  Backbone, Marionette, and backbone.modelbinder.

I've looked through Backbone's source code (https://github.com/jashkenas/backbone/blob/1.1.0/backbone.js).  They never use merge, mergeWith, or defaultsDeep from the underscore dependency.  

I also looked at the Marionette code (https://github.com/marionettejs/backbone.marionette/blob/v2.4.1/lib/backbone.marionette.js).  They never use merge, mergeWith, or defaultsDeep from the underscore dependency.  

I looked at backbone.modelbinder 1.1.0 (https://github.com/theironcook/Backbone.ModelBinder/blob/1.1.0/Backbone.ModelBinder.js) as well.  They never use merge, mergeWith, or defaultsDeep from the underscore dependency.  

For our own code, we pull it in only to use bindAll.  

As such, we aren't affected by this vulnerability.  If we choose to still upgrade, we'll need to go all the way from lodash 2.2.1 to the at least 4.17.11, which is known not to be a simple upgrade path.